Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2838,25 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for
+     * the admin screen.
+     * Setting to false will hide mosaic button for
+     * the admin screen.
+     */
+    public function setIsShowMosaicButton($isShow)
+    {
+        $var = (bool) $isShow;
+
+        if (!$var) {
+            $this->hideMosaicButton();
+
+            return;
+        }
+
+        $this->showMosaicButton();
```

Comment:
Yeah but you could have avoided the `var` [OTV](http://a-nickels-worth.blogspot.fr/2016/04/a-guide-to-naming-variables.html#orgheadline13)